docs/sdk/flutter/initialize.mdx

```diff
@@ -5,50 +5,119 @@ displayed_sidebar: sdk
 description: "Flutter SDK - Initialization | Documentation - Web3Auth"
 ---
 
+import TabItem from "@theme/TabItem";
+import Tabs from "@theme/Tabs";
+
 ---
```

Comment:
The flutter guide has more initialisation and installation steps. Are we not adding them here?

Reply:
Added.